package/MDAnalysis/auxiliary/core.py

```diff
@@ -0,0 +1,33 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+
+from . import _AUXREADERS
+from ..lib import util
+
+def get_auxreader_for(auxdata, format=None):
+    if format is None:
+        if type(auxdata) == str:
```

Comment:
You don't want to check if something is a particular class like this, instead use `isinstance` which also allows for subclassing.

Also, strings are one of the things that got changed in py3 so you need a little six magic here...
```python
from six import string_types

if isinstance(auxdata, string_types):
    do stuff
```